lua/modules/editor/config.lua

```diff
@@ -415,17 +435,18 @@ end
 function config.imselect()
 	-- fcitx5 need a manual config
 	if vim.fn.executable("fcitx5-remote") == 1 then
-		vim.cmd([[
-		let g:im_select_get_im_cmd = ["fcitx5-remote"]
-		let g:im_select_default = '1'
-		let g:ImSelectSetImCmd = {
+		vim.api.nvim_cmd({
+			[[ let g:im_select_get_im_cmd = ["fcitx5-remote"] ]],
+			[[ let g:im_select_default = '1' ]],
```

Comment:
```suggestion
			[[ let g:im_select_get_im_cmd = ["fcitx5-remote"] ]],
			[[ let g:im_select_default = '1'                  ]],
```
let's make them aligned.